src/foam/nanos/crunch/UserCapabilityJunctionRefine.js

```diff
@@ -9,7 +9,7 @@ foam.CLASS({
   name: 'UserCapabilityJunctionRefine',
   refines: 'foam.nanos.crunch.UserCapabilityJunction',
 
-  implements: [ 'foam.nanos.auth.LifecycleAware' ],
+  implements: [ 'foam.nanos.auth.LifecycleAware', 'net.nanopay.liquidity.crunch.TransactionAware' ],//TODO move to NANOPAY project
```

Comment:
I'm sure we refine this on the nanopay side somewhere. 
If we don't we do now.  Please add this refinement on the nanopay side.

Reply:
I will